internal/cri/server/sandbox_stats_list.go

```diff
@@ -54,7 +54,7 @@ func (c *criService) ListPodSandboxStats(
 			default:
 				stats[i] = sandboxStats
 			}
-		}()
+		}(sb)
```

Comment:
stupid nitpick: but wouldn't it be clearer to use the same trick to solve both instances of the same problem, either by doing `i, sb := i, sb` or passing both `i` and `sb` as arguments to the goroutine ?

Reply:
🤝 thanks for the heads-up.